docs/manual/docs/cmd/spo/app/app-add.md

```diff
@@ -24,26 +25,33 @@ Option|Description
 
 ## Remarks
 
-To add an app to the tenant app catalog, you have to first log in to a SharePoint site using the [spo login](../login.md) command, eg. `spo login https://contoso.sharepoint.com`.
+To add an app to the tenant or site collection app catalog, you have to first log in to a SharePoint site using the [spo login](../login.md) command, eg. `spo login https://contoso.sharepoint.com`.
 
 When specifying the path to the app package file you can use both relative and absolute paths. Note, that `~` in the path, will not be resolved and will most likely result in an error.
 
-If you try to upload a package that already exists in the tenant app catalog without specifying the `--overwrite` option, the command will fail with an error stating that the specified package already exists.
+If you try to upload a package that already exists in the app catalog without specifying the `--overwrite` option, the command will fail with an error stating that the specified package already exists.
 
 ## Examples
 
 Add the _spfx.sppkg_ package to the tenant app catalog
 
 ```sh
-spo app add -p /Users/pnp/spfx/sharepoint/solution/spfx.sppkg
+spo app add --filePath /Users/pnp/spfx/sharepoint/solution/spfx.sppkg
 ```
 
 Overwrite the _spfx.sppkg_ package in the tenant app catalog with the newer version
 
 ```sh
-spo app add -p sharepoint/solution/spfx.sppkg --overwrite
+spo app add --filePath sharepoint/solution/spfx.sppkg --overwrite
 ```
 
+Add the _spfx.sppkg_ package to the site collection app catalog of the site you are currencly logged in
```

Comment:
In the CLI we don't use the concept of current site. When we have commands that operate on a particular site, we ask the user to specify the URL in the command arguments. Because authentication is typically interactive, the current format of the command would be impractical, because it would require you to reconnect in the middle of the script if you wanted to add a solution to site collection app catalogs of two different sites. Could we change this command to allow specifying the URL of the site where the site collection app catalog is located? Additionally, could you double check that all other commands follow this pattern?

Reply:
Yes, I will change that shortly. Should I not allow update if site collection app catalog specified, but no siteUrl specified?